packages/preview/README.md

```diff
@@ -0,0 +1,8 @@
+# Theia - Preview Extension
+
+It comes with built-in support for rendering markdown files.
```

Comment:
Sorry, I did not mean to say replace your sentence, but add to it :-)

Reply:
✔️